### DISCUSSION TITLE
网页转APP发生一些BUG

### DISCUSSION THREAD

**Discussion by githudzxl** [AlistGo/alist#6896] [Q&A]
我第一次搞不会用的是哈利提供的版本
hali.life/?p=590

BUG：
播放视频的时候没有全屏需要把手机的竖屏锁定关闭然后
横起来才能看到
竖起来的图片：
![IMG_20240729_005042](https://github.com/user-attachments/assets/71d14d89-763f-4b4a-8ff5-fff9b1eac16a)
横起来的图片：
![IMG_20240729_010133](https://github.com/user-attachments/assets/6caa8f0a-1200-4fe2-8753-b341f927b251)
用的APP转的
![Screenshot_20240729_005114](https://github.com/user-attachments/assets/dfcd0455-f8c0-4b6e-83dd-cc0258973402)

**Comment by githudzxl** [comment]
我应该怎么做

**Comment by Simonzhang8** [comment]
> BUG： 播放视频的时候没有全屏需要把手机的竖屏锁定关闭然后 横起来才能看到

点一下视频画面控制条就出来了

**Reply by githudzxl** [reply]
这是视频：
https://github.com/user-attachments/assets/d0753dce-be70-4499-93c5-f18f6e65ccbe

**Reply by githudzxl** [reply]
感谢回答但是并不是解决方案

**Comment by Simonzhang8** [comment]
看了下你的视频，你这个是手机屏幕比例的问题，暂时好像没有什么很好的解决方法😭
